src/components/QuizGame/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { fetchQuizQuestions } from "../../components/QuizGameApi";
+import "antd/dist/antd.css";
+import QuestionCard from "../../components/QuizGameQuestions";
+import "./index.css";
+const Difficulty = {
+  EASY: "easy",
+  MEDIUM: "medium",
+  HARD: "hard",
+};
+const TOTAL_QUESTIONS = 10;
+const QuizGame = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [questions, setQuestions] = useState([]);
+  const [currentQuestionNumber, setCurrentQuestionNumber] = useState(0);
+  const [userAnswers, setUserAnswers] = useState([]);
+  const [score, setScore] = useState(0);
+  const [isGameOver, setIsGameOver] = useState(true);
```

Comment:
i didn't get the question!

Reply:
it should definitely start as false.

Reply:
@louisrli  actually it  is started with true for designing reason  no question will be displayed to the dom until you click on start button , i will send a screen shot

Reply:
I'm not saying it doesn't work. I'm saying it doesn't make sense. I would ask the same thing tarkan is saying. if I told you in English, "it is true that the game is over before it starts", this would not make sense.

Reply:
@louisrli  so what the solution you would suggest, should i add a comment or fix it so the game will start directly?

Reply:
i edited it and added a new state it start with false now